Functions/Example/Tests/FUNSerializerTests.m

```diff
@@ -191,10 +196,15 @@ - (void)testEncodeMap {
     @"foo" : @1,
     @"bar" : @"hello",
     @"baz" : @[
-      @3, @{
-        @"@type" : @"type.googleapis.com/google.protobuf.Int64Value",
-        @"value" : @"4",
-      }
+        @3,
```

Comment:
I imagine the point of this test is to establish that we handle 64-bit values properly. As a result, I bet that a better way to fix this would be to ensure that the input results in a 64-bit value regardless of the runtime. I think you could accomplish this by storing a value larger than 32-bits and using an `LL` suffix.

Reply:
The `LL` suffix isn't sufficient, but making the value big enough forces it to be a long long NSNumber on 32 bit also.